tests/posix/headers/src/signal_h.c

```diff
@@ -40,12 +40,12 @@ ZTEST(posix_headers, test_signal_h)
 	zassert_not_equal(-1, offsetof(union sigval, sival_int));
 	zassert_not_equal(-1, offsetof(union sigval, sival_ptr));
 
-	/* zassert_not_equal(-1, RTSIG_MAX); */ /* not implemented */
-	/* zassert_true(SIGRTMAX - SIGRTMIN >= RTSIG_MAX); */ /* not implemented */
+	zassert_not_equal(-1, RTSIG_MAX);
+	zassert_true(SIGRTMAX - SIGRTMIN >= RTSIG_MAX);
```

Comment:
should this be strictly equal?

Reply:
> The range SIGRTMIN through SIGRTMAX inclusive shall include at least {RTSIG_MAX} signal numbers.

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html